walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedIdClaim.kt

```diff
@@ -9,12 +9,15 @@ package com.microsoft.walletlibrary.verifiedid
  * Represents each claim that is embedded in a VerifiedID.
  */
 data class VerifiedIdClaim(
-    // Represents name of the claim.
+    // Represents id of the claim.
     val id: String,
```

Comment:
Why do we need to separate label and id?

Reply:
the `id` for claims is the field name within the credential (ie `givenName`,) while the label is a localized string (`Given Name`.)